src/app/views/templates/sms-verification-disabled-admin.server.view.html

```diff
@@ -4,20 +4,21 @@
   <body>
     <p>Dear form admin,</p>
     <p>
-      You are receiving this email as you own forms with SMS OTP verification
-      enabled:
+      You own forms with free SMS OTP verification on a Mobile Number field:
     </p>
     <ol>
       <% forms.forEach(function({title, link}) { %>
       <li><a href="<%= link %>"> <%= title %> </a></li>
       <% }) %>
     </ol>
     <p>
-      As you have reached the free tier limit of <%= smsVerificationLimit %>
-      verifications sent, SMS OTP verification has been
-      <b>automatically disabled</b> on all your form(s). Respondents will still
-      be able to submit to your form(s) but will not be prompted to verify their
-      mobile numbers.
+      <b>
+        As you have reached the free tier limit of <%= smsVerificationLimit %>
+        per account, SMS OTP verification has been automatically disabled on all
+        forms you own.
```

Comment:
I thought SMS OTP verification will only be disabled on forms that are NOT onboarded yet? Not all forms they own

Reply:
thx 4 catch, fixed!